slam-ws/src/load_waypoints/scripts/load_waypoints_server.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+
+import rospy, os, json 
+from load_waypoints.srv import *
+
+all_waypoints = list()
+
+def parse_json():
+    base_dir = rospy.get_param('~arg_name') # ~ added to arg_name because private param 
+    
+    with open(base_dir + '/scripts/waypoints.json') as f:
+        data = json.load(f)
```

Comment:
What happens if the file is not a valid JSON? We should make sure our server exits gracefully.